internal/eth/txn_test.go

```diff
@@ -23,6 +23,7 @@ import (
 	"reflect"
 	"testing"
 
+	"github.com/ethereum/go-ethereum/common/hexutil"
```

Comment:
Due to the licensing of the `github.com/ethereum/go-ethereum` we cannot depend on it in any code in this repo.